agent/exploits/cve_2024_50550.py

```diff
@@ -0,0 +1,39 @@
+"""Agent Asteroid implementation for CVE-2024-50550"""
+
+import re
+
+from packaging import version
+
+from agent import definitions
+from agent import exploits_registry
+from agent.exploits import webexploit
+
+VERSION_PATTERN = re.compile(r"Stable tag:\s(\d+\.\d+\.\d+)")
+MAX_VULNERABLE_VERSION = version.parse("6.5.1")
+VULNERABILITY_TITLE = "LiteSpeed Cache Vulnerability Allowing privilege escalation"
+VULNERABILITY_REFERENCE = "CVE-2024-50550"
+VULNERABILITY_DESCRIPTION = (
+    "Incorrect Privilege Assignment vulnerability in LiteSpeed Technologies LiteSpeed Cache allows Privilege Escalation."
+    "This issue affects LiteSpeed Cache versions before 6.5.2."
+)
+RISK_RATING = "HIGH"
+
+
+@exploits_registry.register
+class CVE202450550Exploit(webexploit.WebExploit):
+    accept_request = definitions.Request(
+        method="GET", path="/wp-content/plugins/litespeed-cache/readme.txt"
+    )
+    check_request = definitions.Request(
+        method="GET", path="/wp-content/plugins/litespeed-cache/readme.txt"
```

Comment:
Path can be const!!

Reply:
The occurrences of this path is 5 times. I would put it into a const too.

Reply:
it's not always the same path for both accept and check so it's more clear this way!